Add check for CSV file headers or avoid exception

If I have a file with a header like this for users create with CSV, I get an exception:

```bash
panic: runtime error: invalid memory address or nil pointer dereference
```

--- a/cmd/csv.go
+++ b/cmd/csv.go
@@ -17,6 +17,7 @@ limitations under the License.
 package cmd
 
 import (
+	"bufio"
 	"fmt"
 	"os"
 
@@ -26,7 +27,28 @@ import (
 	"github.com/spf13/viper"
 )
 
-func readUsersFromCSV(jsonOutput bool) ([]*models.User, error) {
+func checkCSVHeader(header string) error {
+	file := viper.GetString("file")
+	csvFile, err := os.Open(file)
+	if err != nil {
+		return fmt.Errorf("can't open CSV file")
+	}
+	defer csvFile.Close()
+
+	// Try to read first row and check if row is valid
+	reader := bufio.NewReader(csvFile)
+	line, _, err := reader.ReadLine()
+	if err != nil {
+		return err
+	}
+
+	if header != string(line) {
+		return fmt.Errorf("wrong header")
+	}
+	return nil
+}
+
+func readUsersFromCSV(jsonOutput bool, header string) ([]*models.User, error) {
 	// Read and open file
 	file := viper.GetString("file")
 	csvFile, err := os.Open(file)
@@ -34,6 +56,12 @@ func readUsersFromCSV(jsonOutput bool) ([]*models.User, error) {
 		return nil, fmt.Errorf("can't open CSV file")
 	}
 	defer csvFile.Close()
+
+	// Try to read first row and check if row is valid
+	err = checkCSVHeader(header)
+	if err != nil {
+		return nil, err
+	}
 
 	// Try to unmarshal CSV file usin gocsv
 	users := []*models.User{}
@@ -43,7 +71,7 @@ func readUsersFromCSV(jsonOutput bool) ([]*models.User, error) {
 	return users, nil
 }
 
-func readGroupsFromCSV(jsonOutput bool) ([]*models.Group, error) {
+func readGroupsFromCSV(jsonOutput bool, header string) ([]*models.Group, error) {
 	// Read and open file
 	file := viper.GetString("file")
 	csvFile, err := os.Open(file)
@@ -51,6 +79,12 @@ func readGroupsFromCSV(jsonOutput bool) ([]*models.Group, error) {
 		return nil, fmt.Errorf("can't open CSV file")
 	}
 	defer csvFile.Close()
+
+	// Try to read first row and check if row is valid
+	err = checkCSVHeader(header)
+	if err != nil {
+		return nil, err
+	}
 
 	// Try to unmarshal CSV file usin gocsv
 	groups := []*models.Group{}

--- a/cmd/csv_create_groups.go
+++ b/cmd/csv_create_groups.go
@@ -40,7 +40,7 @@ func CsvCreateGroupsCmd() *cobra.Command {
 			messages := []string{}
 
 			// Read and open file
-			groups, err := readGroupsFromCSV(jsonOutput)
+			groups, err := readGroupsFromCSV(jsonOutput, "name,description,members")
 			if err != nil {
 				return err
 			}

--- a/cmd/csv_create_groups_test.go
+++ b/cmd/csv_create_groups_test.go
@@ -153,7 +153,7 @@ func TestCsvCreateGroups(t *testing.T) {
 			name:           "file with wrong header",
 			cmd:            CsvCreateGroupsCmd(),
 			args:           []string{"--server", "http://127.0.0.1:50033", "--file", "/tmp/file3.csv"},
-			errorMessage:   "record on line 2: wrong number of fields",
+			errorMessage:   "wrong header",
 			successMessage: "",
 		},
 		{

--- a/cmd/csv_create_users.go
+++ b/cmd/csv_create_users.go
@@ -43,7 +43,7 @@ func CsvCreateUsersCmd() *cobra.Command {
 			messages := []string{}
 
 			// Read and open file
-			users, err := readUsersFromCSV(jsonOutput)
+			users, err := readUsersFromCSV(jsonOutput, "username,firstname,lastname,email,password,ssh_public_key,jpeg_photo,manager,readonly,locked,groups")
 			if err != nil {
 				return err
 			}

--- a/cmd/csv_create_users_test.go
+++ b/cmd/csv_create_users_test.go
@@ -168,7 +168,7 @@ func TestCsvCreateUsers(t *testing.T) {
 			name:           "file with wrong header",
 			cmd:            CsvCreateUsersCmd(),
 			args:           []string{"--server", "http://127.0.0.1:50032", "--file", "/tmp/file3.csv"},
-			errorMessage:   "record on line 2: wrong number of fields",
+			errorMessage:   "wrong header",
 			successMessage: "",
 		},
 		{

--- a/cmd/csv_delete_groups.go
+++ b/cmd/csv_delete_groups.go
@@ -41,7 +41,7 @@ func CsvDeleteGroupsCmd() *cobra.Command {
 			messages := []string{}
 
 			// Read and open file
-			groups, err := readGroupsFromCSV(jsonOutput)
+			groups, err := readGroupsFromCSV(jsonOutput, "gid, name")
 			if err != nil {
 				return err
 			}

--- a/cmd/csv_delete_groups_test.go
+++ b/cmd/csv_delete_groups_test.go
@@ -190,7 +190,7 @@ func TestCsvDeleteGroups(t *testing.T) {
 			name:           "file with wrong header",
 			cmd:            CsvDeleteGroupsCmd(),
 			args:           []string{"--server", "http://127.0.0.1:50034", "--file", "/tmp/file3.csv"},
-			errorMessage:   "record on line 2: wrong number of fields",
+			errorMessage:   "wrong header",
 			successMessage: "",
 		},
 		{

--- a/cmd/csv_delete_users.go
+++ b/cmd/csv_delete_users.go
@@ -40,7 +40,7 @@ func CsvDeleteUsersCmd() *cobra.Command {
 			jsonOutput := viper.GetBool("json")
 
 			// Read and open file
-			users, err := readUsersFromCSV(jsonOutput)
+			users, err := readUsersFromCSV(jsonOutput, "uid, username")
 			if err != nil {
 				return err
 			}

--- a/cmd/csv_delete_users_test.go
+++ b/cmd/csv_delete_users_test.go
@@ -161,7 +161,7 @@ func TestCsvDeleteUsers(t *testing.T) {
 			name:           "file with wrong header",
 			cmd:            CsvDeleteUsersCmd(),
 			args:           []string{"--server", "http://127.0.0.1:50035", "--file", "/tmp/file3.csv"},
-			errorMessage:   "record on line 2: wrong number of fields",
+			errorMessage:   "wrong header",
 			successMessage: "",
 		},
 		{

